utils/reddit.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+import nltk
+import pandas as pd
+import praw
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
+
+# retrieve env vars
+CLIENT_ID = os.environ.get("REDDIT_CLIENT_ID")
```

Comment:
@majulahsingapuri 

is it better if we do:

```python
from config import config
CLIENT_ID = config.reddit_client_id
CLIENT_SECRET = config.reddit_client_secret
USER_AGENT = config.reddit_user_agent
```

Reply:
Oh yes, must have submitted an old snippet there by mistake